OOPExercises/Student.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OOPExercises
+{
+    class Student
+    {
+        private string name;
+        private DateTime birthday;
+        private double score;
+
+        public string Name
```

Comment:
Change to auto property and remove the private fields
```C#
public string Name{set;get;}
```
Do it for all properties

Reply:
Done